From: Stefano Rivera <stefanor@debian.org>
Date: Thu, 8 Oct 2015 10:23:38 -0700
Subject: Use Debian's ply instead of the bundled ply

We prefer to avoid bundled libraries in Debian, don't use the bundled
ply.

Last-Update: 2013-01-12
Forwarded: not-needed
---
 pycparser/c_lexer.py  | 4 ++--
 pycparser/c_parser.py | 2 +-
 setup.py              | 2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/pycparser/c_lexer.py b/pycparser/c_lexer.py
index 22c64bc..a97c5d1 100644
--- a/pycparser/c_lexer.py
+++ b/pycparser/c_lexer.py
@@ -8,8 +8,8 @@
 #------------------------------------------------------------------------------
 import re
 
-from .ply import lex
-from .ply.lex import TOKEN
+from ply import lex
+from ply.lex import TOKEN
 
 
 class CLexer(object):
diff --git a/pycparser/c_parser.py b/pycparser/c_parser.py
index d31574a..5fc3544 100644
--- a/pycparser/c_parser.py
+++ b/pycparser/c_parser.py
@@ -6,7 +6,7 @@
 # Eli Bendersky [https://eli.thegreenplace.net/]
 # License: BSD
 #------------------------------------------------------------------------------
-from .ply import yacc
+from ply import yacc
 
 from . import c_ast
 from .c_lexer import CLexer
diff --git a/setup.py b/setup.py
index f7e89bf..4ffc339 100644
--- a/setup.py
+++ b/setup.py
@@ -60,7 +60,7 @@ setup(
         'Programming Language :: Python :: 3.12',
     ],
     python_requires=">=3.8",
-    packages=['pycparser', 'pycparser.ply'],
+    packages=['pycparser'],
     package_data={'pycparser': ['*.cfg']},
     cmdclass={'install': install, 'sdist': sdist},
 )
